py/desitarget/mock/mockmaker.py

```diff
@@ -3343,11 +3355,12 @@ def make_spectra(self, data=None, indx=None, seed=None, no_spectra=False):
             # Gather all the files containing at least one QSO skewer.
             alllyafile = data['LYAFILES'][indx]
             uniquelyafiles = sorted(set(alllyafile))
-
             for lyafile in uniquelyafiles:
                 these = np.where( alllyafile == lyafile )[0]
 
                 mockid_in_data = data['MOCKID'][indx][these]
+                if not os.path.isfile(lyafile):
+                    lyafile=lyafile.replace(".gz","")
                 mockid_in_mock = (fitsio.read(lyafile, columns=['MOCKID'], upper=True,
```

Comment:
I guess some skewers are gzipped and others are not? Should we raise an exception if neither type of file is found, or just let the code crash and burn?

Reply:
Versions older that v5 were not gziped... I don't think this will be used much, anyways... so   I think is better to  add the exception to catch other possibilities...